apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeSection.tsx

```diff
@@ -1,22 +1,27 @@
 import { useEffect, useState } from 'react';
 import { Paragraph, Stack, Subheading } from '@contentful/f36-components';
-import { AllContentTypes, AllContentTypeEntries, ContentTypeEntries } from 'types';
+import {
+  AllContentTypes,
+  AllContentTypeEntries,
+  ContentTypeEntries,
+  ContentTypes,
+  ContentTypeValue,
+} from 'types';
 import AssignContentTypeCard from 'components/config-screen/assign-content-type/AssignContentTypeCard';
 import sortBy from 'lodash/sortBy';
-import { useSDK } from '@contentful/react-apps-toolkit';
+import { ContentTypeProps, createClient, KeyValueMap } from 'contentful-management';
 import { AppExtensionSDK } from '@contentful/app-sdk';
-import { ContentTypeProps, createClient } from 'contentful-management';
-import useKeyService from 'hooks/useKeyService';
-
-const AssignContentTypeSection = () => {
-  const {
-    contentTypes,
-    handleContentTypeChange,
-    handleContentTypeFieldChange,
-    handleAddContentType,
-    handleRemoveContentType,
-  } = useKeyService({});
+import { useSDK } from '@contentful/react-apps-toolkit';
+interface Props {
+  mergeSdkParameters: Function;
+  onIsValidContentTypeAssignment: Function;
+  parameters: KeyValueMap;
+}
+
+const AssignContentTypeSection = (props: Props) => {
+  const { mergeSdkParameters, onIsValidContentTypeAssignment } = props;
```

Comment:
We should also destructure the `parameters` prop and this component should have a `useEffect` where the `parameters.contentTypes` is setting the `contentTypes` state. Right now, the saved `contentTypes` in the installation parameters are not being rendered.